test/display_tests/display.jl

```diff
@@ -12,15 +12,13 @@ setprecision(Interval, Float64)
         a = 1..2
         b = -1.1..1.3
         c = Interval(pi)
-        d = @interval(π)
```

Comment:
Is there any difference in using `Interval(pi)` or `@interval(π)`? If not, why not simply test `Interval(pi) == @interval(π)`?

Reply:
`@interval(π)` doesn't use the fancy new precomputed stuff, but should return the same value and `Interval(pi) == @interval(π)` is indeed tested below.

I'm not sure I can do the same trick I did on the constructor on the macro, but I'll have a look.

Reply:
I tried to make `atomic` use the `Interval` constructor, but then trying to use `IntervalArithmetic` crashed Julia... I don't know what's going on, so for now I'll let it as it is.

Reply:
Sorry, I didn't notice that the tests were already there...